src/v2/controllers/FormController.js

```diff
@@ -31,6 +31,14 @@ export default BaseLoginController.extend({
   postRender () {
     const currentViewState = this.options.appState.getCurrentViewState();
 
+    if (currentViewState.name === 'select-factor') {
```

Comment:
prefer to move this logic into `getCurrentViewState` and add comments regarding why override the form name.